widgets/pulseaudio.lua

```diff
@@ -24,7 +24,14 @@ local function factory(args)
    local settings    = args.settings or function() end
    local scallback   = args.scallback
 
-   pulseaudio.cmd    = args.cmd or "pacmd list-" .. devicetype .. "s | sed -n -e '0,/*/d' -e '/base volume/d' -e '/volume:/p' -e '/muted:/p' -e '/device\\.string/p'"
+   pulseaudio.cmd    = args.cmd or ("pacmd list-" .. devicetype .. "s | sed -n -e '/* index:/,/index:/ !d' -e '/* index:/ p' -e '/base volume:/ d' -e '/volume:/ p' -e '/muted:/ p' -e '/device\\.string/ p'")
```

Comment:
Alright, will move the comment to the wiki.

How about the change from `-e '0,/*/d'` to `-e '/* index:/,/index:/ !d' -e '/* index:/ p'`? Are you fine with that? I found it a bit clearer to understand and this will also strip all device sections after the default one. But I'm not that familiar with sed and don't know if this is robust.